### PR TITLE
GEOMESA-1365 Allowing for configuration of individual indices

### PR DESCRIPTION
**Pull request by elahrvivaz**
* New tools command to add/update individual indices
* New m/r job to back-fill individual indices

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
need to update docs

**Comment by jahhulbert-ccri** [comment]
so far looks good. More tables with built in record ID! SAVE THE BYTES! ALL THE BYTES!

**Comment by jahhulbert-ccri** [comment]
looks good...figure i can do some testing at some point

File: docs/user/data_management.rst [hunk 1]
```diff
@@ -276,6 +276,123 @@ features), you must explicitly enable "mixed" indexing mode with ``geomesa.mixed
...
++-----------+---------+----------------------------------------------------------------------------+
+| Record    | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+| Attribute | 2       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+
+
+Schema version 8 - 1.2.2 through 1.2.4
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```
Comment (mdzimmerman) [inline docs/user/data_management.rst]: Maybe add a note above the table saying that the GeoHash index was removed?

Comment (elahrvivaz) [inline docs/user/data_management.rst]: I put a note by Z2 that it replaces the geohash index - was it not clear though?

**Pull request merged**